api/src/pcapi/routes/serialization/collective_offers_serialize.py

```diff
@@ -39,6 +39,14 @@
 )
 
 
+def validate_venue_id(venue_id: int | str | None) -> int | None:
+    # TODO(jeremieb): remove this validator once there is no empty
+    # string stored as a venueId
+    if not venue_id:
+        return None
+    return int(venue_id)  # should not be needed but it makes mypy happy
```

Comment:
On déclare venue_id possiblement comme une string (pas forcément vide du point de vue de mypy)

Reply:
As-tu une idée de comment lui dire que si c'est une _string_, ce sera forcément `""` ?

Reply:
Je rebondissais seulement sur ton commentaire, imo tu peux laisser comme ça, de toute manière comme tu l’indiques le validateur n’est pas amené. Sinon peut-être qu’un `assert not isinstance(venue_id, str)` pourrait marcher ? @jeremieb-pass

Reply:
Si ça te va comme ça, je _merge_ en l'état. Ceci dit, je n'ai pas de préférence pour l'une ou l'autre option.